pkg/controller/cloneset/cloneset_status.go

```diff
@@ -101,4 +102,12 @@ func (r *realStatusUpdater) calculateStatus(cs *appsv1alpha1.CloneSet, newStatus
 	if newStatus.UpdatedReplicas == newStatus.Replicas {
 		newStatus.CurrentRevision = newStatus.UpdateRevision
 	}
+
+	if newStatus.UpdateRevision == newStatus.CurrentRevision {
+		newStatus.ExpectedUpdatedReplicas = 0
```

Comment:
if UpdateRevision= CurrentRevision, ExpectedUpdatedReplicas=replicas sounds more reasonable to me